docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
 
   spid-testenv2:
     container_name: spid-testenv2
-    image: italia/spid-testenv2:latest
+    image: italia/spid-testenv2:fix-c14n-issue-169
```

Comment:
do we need to change this in the future, if that's the case please add a story to remind us